processing/src/main/java/io/druid/query/timeboundary/TimeBoundaryQuery.java

```diff
@@ -52,13 +53,15 @@
 
   private static final byte CACHE_TYPE_ID = 0x0;
 
+  private final DimFilter dimFilter;
   private final String bound;
 
   @JsonCreator
   public TimeBoundaryQuery(
       @JsonProperty("dataSource") DataSource dataSource,
       @JsonProperty("intervals") QuerySegmentSpec querySegmentSpec,
       @JsonProperty("bound") String bound,
+      @JsonProperty("filter") DimFilter dimFilter,
```

Comment:
This requires updating the timeboundary query documentation in docs/content/querying